client.go

```diff
@@ -65,8 +66,9 @@ var (
 // Client performs the client side part of swaps. This interface exists to be
 // able to implement a stub.
 type Client struct {
-	started uint32 // To be used atomically.
-	errChan chan error
+	started      uint32 // To be used atomically.
+	errChan      chan error
+	abandonChans map[lntypes.Hash]chan struct{}
```

Comment:
I think this justifies a comment since it's not trivial to understand what we store in the map and why.